contents/javascript/VueJs/Intro.md

```diff
@@ -0,0 +1,85 @@
+# VueJs
+
+Author: Lu Lechuan
+
+A JavaScript framework is an application framework written in JavaScript. It differs from a JavaScript library in its control flow: a library offers functions to be called by its parent code, whereas a framework defines the entire application design.
+JavaScript frameworks aim to improve code quality and maintainablity, example of some javascript frameworks include Angular, React and Vue.
```

Comment:
"example of some javascript framework include..." should be "some examples of javacript framework are ..."